README.md

```diff
@@ -227,57 +227,99 @@ _Default to `null`._
 
 ## Usage in Symfony
 
-### Client as a service
+### Configuration
 
-Just declare the proper service in `services.yaml`:
+You'll need to add the bundle in `bundles.php`:
+
+```php
+// config/bundles.php
+return [
+    // ...
+    JoliCode\Elastically\Bridge\Symfony\ElasticallyBundle::class => ['all' => true],
+];
+```
+
+Then configure the bundle:
 
 ```yaml
-services:
-    JoliCode\Elastically\IndexNameMapper:
-        arguments:
-            $prefix: null # or a string to prefix index name
-            $indexClassMapping:
-                indexName: My\AwesomeDTO
+# config/packages/elastically.yaml
+elastically:
+    connections:
+        default:
+            client:
+                host:                '%env(ELASTICSEARCH_HOST)%'
+                # If you want to use the Symfony Http Client:
+                transport:           '@JoliCode\Elastically\Transport\HttpClientTransport'
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I think that is should be the default in a Symfony context.

Reply:
What if the http-client is not installed, or installed but not registered?
And this configuration is far from perfect IMHO, because it use the default HttpClient, and the default client can have some very special setting that might not work with ES. (header, base_uri, etc)